common/djangoapps/student/tests/test_roles.py

```diff
@@ -189,7 +192,8 @@ def test_get_orgs_for_user(self):
 @ddt.ddt
 class RoleCacheTestCase(TestCase):  # lint-amnesty, pylint: disable=missing-class-docstring
 
-    IN_KEY = CourseKey.from_string('edX/toy/2012_Fall')
+    IN_KEY_STRING = 'edX/toy/2012_Fall'
```

Comment:
I realize that you're just copying what's there, but this type of Course ID doesn't work for anything in the system any longer. While it should make no difference for these tests specifically, please use the newer style course keys, e.g. `course-v1:edX+toy+2012_Fall`).

I thought we had managed to kill most of these IDs from the test suite, except where we had to keep them for backwards compatibility.

Reply:
(The library key you're using is fine, fwiw.)

Reply:
done